core/templates/dev/head/pages/exploration_editor/feedback_tab/ThreadDataService.js

```diff
@@ -106,9 +108,11 @@ oppia.factory('ThreadDataService', [
       $http.get(_THREAD_HANDLER_PREFIX + threadId).then(function(response) {
         var allThreads = _data.feedbackThreads.concat(_data.suggestionThreads);
         for (var i = 0; i < allThreads.length; i++) {
-          if (allThreads[i].thread_id === threadId) {
+          if (allThreads[i].threadId === threadId) {
             allThreads[i].messages = response.data.messages;
```

Comment:
We shouldn't just be setting properties on threads willy-nilly. It doesn't seem like this is an actual property defined in the thread domain objects? Should it be?

Reply:
Yeah. Added.